pkg/frontend/admin_supportedvmsizes_list_test.go

```diff
@@ -0,0 +1,76 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/validate"
+	"github.com/Azure/ARO-RP/pkg/metrics/noop"
+)
+
+func TestSupportedvmsizes(t *testing.T) {
+	ctx := context.Background()
+	mastervmsizes, _ := json.MarshalIndent(validate.SupportedMasterVmSizes, "", "    ")
+	workervmsizes, _ := json.MarshalIndent(validate.SupportedMasterVmSizes, "", "    ")
```

Comment:
```suggestion
	workervmsizes, _ := json.MarshalIndent(validate.SupportedWorkerVmSizes, "", "    ")
```
Right now we are testing the master path, but not the worker path.

Reply:
Also, why no error checking here?

Reply:
Added the error check, and corrected the worker path